src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPluginExtension.kt

```diff
@@ -55,4 +57,27 @@ open class MavenPublishPluginExtension(project: Project) {
   var repositoryPassword: String?
     get() = defaultTarget.repositoryPassword
     set(value) { defaultTarget.repositoryPassword = value }
+
+  /**
+   * Allows to add additional [MavenPublishTargets][MavenPublishTarget] to publish to multiple repositories.
```

Comment:
`[MavenPublishTargets][MavenPublishTarget]` isn't one enough?

Reply:
The first one is what you see in when the markdown is rendered the second is the link to the class. I personally prefer that over writing `[MavenPublishTarget]s`